### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by l-arnold** [l-arnold/tkl-nomadic-odoo#30]
There is a fair bit of information we should Impart.

Thinking that this may be a "moving" subject, can we have a WIKI?
I otherwise propse we build it into the Documentation system with in var/www here
(see port https://Server:12324)

**Comment by JedMeister** [comment]
Oops probably should have posted here... 

Git repos have wikis too (as it seems [you've already realised!](https://github.com/l-arnold/tkl-nomadic-odoo/wiki)); although IMO keeping it as a text file in the docs dir is better in the longer term... 

FWIW Alon and I were talking about this the other day and thinking that we might move all appliance specific docs to a docs dir within each appliance's build code (like you have in Odoo). That way the docs can be tracked over time and relevant to each release. Currently the TurnKey appliance build code is tagged with the release version so generally speaking viewing the docs at that tag point will give the docs relevant to that version...

**Comment by l-arnold** [comment]
I have built first Drafts of 4 Documents in the Root Docs File now.
Should be enough to get folks started.  Some Art would be nice but we are not writing HTML there.

**Comment by l-arnold** [comment]
I would like a "web driven" but easy to navigate system for "internal/external" documentation. 
Best to use Dreamweaver?  Should some sort of internal wiki (very lightweight) be integrated?

Thinking out loud as is too often the case.

**Comment by DocCyblade** [comment]
Thinking out loud as well... Is there not a apache2 module that would convert RST(Restructured Text) or MD(Mark Down) files to HTML? Like GIT does? This way you could just store them in text format MD or RST and have apache serve them up as is. No need to know HTML. The pages would be basic but really thats all you need any way.

**Comment by JedMeister** [comment]
> Thinking out loud as well... Is there not a apache2 module that would convert RST(Restructured Text) or MD(Mark Down) files to HTML? Like GIT does? This way you could just store them in text format MD or RST and have apache serve them up as is. No need to know HTML. The pages would be basic but really thats all you need any way.

If there isn't there should be!

[update]
Not quite an Apache module but might be of value?:
http://linux.die.net/man/1/markdown

**Comment by l-arnold** [comment]
yeah.  I was thinking that too.  It seems thate perhaps Odoo supports it.  For sure Git supports it.

I threw some in there and if people want to read them nice they can go to the GitHub home for the files.  One solution for now.

**Comment by l-arnold** [comment]
Docs (somewhat Stranded) in "dev" branch until we get Dev Branch booting into Odoo.

**Comment by l-arnold** [comment]
Closing for Now.  Will Reopen when working on a new build.
